habit/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import * as store from './store';
+import {Provider} from 'react-redux';
+import App from '../App';
+
+export default class AppProvider extends Component {
```

Comment:
This is not connected to the outside (main) index.js and app.js.
So, this code will not do anything in itself.

Need to fix the connection of the store to the App.js in the app

Reply:
there is a connection, the main app is imported and used as a component in this file.